include/Contact.php

```diff
@@ -683,4 +683,47 @@ function formatted_location($profile) {
 
 	return $location;
 }
+
+/**
+ * @brief Returns the account type name
+ *
+ * The function can be called with either the user or the contact array
+ *
+ * @param array $contact contact or user array
+ */
+function account_type($contact) {
+
+	// There are several fields that indicate that the contact or user is a forum
```

Comment:
I know I'm really annoying but I would prefer that you write 1 - 2 more sentences which describe why we have that much fields so it wouldn't be only understandable for insiders which are familiar with the friendica history